diff -ruN ddclient-3.9.0/Makefile ddclient/Makefile
--- ddclient-3.9.0/Makefile	2020-10-24 01:10:25.087344000 -0700
+++ ddclient/Makefile	2020-10-24 01:10:37.600138000 -0700
@@ -3,7 +3,7 @@
 
 PORTNAME=	ddclient
 DISTVERSIONPREFIX=	v
-DISTVERSION=	3.9.0
+DISTVERSION=	3.9.1
 CATEGORIES=	dns
 
 MAINTAINER=	mjl@luckie.org.nz
diff -ruN ddclient-3.9.0/distinfo ddclient/distinfo
--- ddclient-3.9.0/distinfo	2020-10-24 01:10:25.086826000 -0700
+++ ddclient/distinfo	2020-10-24 01:10:45.082573000 -0700
@@ -1,3 +1,3 @@
-TIMESTAMP = 1574324843
-SHA256 (ddclient-ddclient-v3.9.0_GH0.tar.gz) = 9c4ae902742e8a37790d3cc8fad4e5b0f38154c76bba3643f4423d8f96829e3b
-SIZE (ddclient-ddclient-v3.9.0_GH0.tar.gz) = 59018
+TIMESTAMP = 1603527045
+SHA256 (ddclient-ddclient-v3.9.1_GH0.tar.gz) = e4969e15cc491fc52bdcd649d4c2b0e4b1bf0c9f9dba23471c634871acc52470
+SIZE (ddclient-ddclient-v3.9.1_GH0.tar.gz) = 63469
